src/index.html

```diff
@@ -70,8 +81,9 @@ <h2 class="navbar__subtitle">a travel journal</h2>
         class="form__input form__input--country"
       />
       <!-- add photo field-->
-      <label class="form__label form__label--url" id="url">URL</label>
-      <input type="url" id="inputImg" class="form__input form__input--url" />
+      <label class="form__label form__label--url" id="url">Photo</label>
+      <input type="file" id="inputImg" class="form__input form__input--file" />
```

Comment:
A label must be connected to a input always. A way to do it is using `for` attribute with a value matching the input id. For example:

```html
<label class="form__label form__label--url" for="inputImg">Photo</label>
<input type="file" id="inputImg" class="form__input form__input--file" />
```

I noticed you did it in the checkbox but the id and for values do not match. They have too.

There are [other ways](https://www.w3.org/WAI/tutorials/forms/labels/). Personally I like _Associating labels implicitly_ because we don't need to use `for` in that case.